solutions/observability/synthetics/monitor-resources-on-private-networks.md

```diff
@@ -56,42 +56,63 @@ By default {{private-location}}s are configured to allow two simultaneous browse
 
 After setting up {{fleet}}, you’ll connect {{fleet}} to the {{stack}} or your Observability Serverless project and enroll an {{agent}} in {{fleet}}.
 
-Elastic provides Docker images that you can use to run {{fleet}} and an {{agent}} more easily. For monitors running on {{private-location}}s, you *must* use the `elastic-agent-complete` Docker image to create a self-hosted {{agent}} node. The standard {{ecloud}} or self-hosted {{agent}} will not work.
-
+Elastic provides Docker images that you can use to run {{fleet}} and an {{agent}} more easily. For running browser monitors on {{private-location}}s, you *must* use one of the `elastic-agent-complete` Docker image variants, as it includes the required dependencies. The standard {{agent}} variant only supports TCP, ICMP and HTTP monitors.
 ::::{important}
-The `elastic-agent-complete` Docker image is the only way to have all available options that you see in the UI.
+The `elastic-agent` Docker image variants are the only way to have all available options that you see in the UI.
```

Comment:
```suggestion
The `elastic-agent-complete` Docker image is the only way to have all available options that you see in the UI. You can use `elastic-agent` Docker image only in case you will only use lightweight monitors.
```

This is misleading to the eyes of a non-expert user.